core/src/main/scala/org/bykn/bosatsu/Evaluation.scala

```diff
@@ -396,6 +437,8 @@ case class Evaluation[T](pm: PackageMap.Typed[T], externals: Externals) {
             val dt = definedForCons(pc)
             val itemFns = items.map(maybeBind(_))
 
+            val itemsWild = items.forall(isTotalNonBinding(_))
+
             def processArgs(as: List[Value], acc: Env): Option[Env] = {
               // manually write out foldM hoping for performance improvements
```

Comment:
I know this isn't new but it occurred to me it might help readability to have the non-optimal `fold` as a comment